### ISSUE TITLE
new big RC file, try it out

### ISSUE THREAD

**Issue by yrmvgh** [jeremygurr/dcssca#442]
Do you use elliptic, gammafunk, or HDA's rc files? Are you tired of oddities that arise from occasional, mostly food-related incompatibilities? Want an rc file that was designed with Circus Animals in mind? You're in luck... maybe!

I finally rolled my own RC file from exhaustively poring through https://github.com/crawl/crawl/blob/master/crawl-ref/docs/options_guide.txt and lifting some LUA functions from other users. Try it out and let me know whether it has any issues!

https://github.com/yrmvgh/Great-Repository-Name/blob/master/Primary.rc.lua

**Comment by yrmvgh** [comment]
_(The Kobold)_ can you summarize your .rc?
- (many years pass by)

_(The Villain)_ I guess the summary would be, a vast array of convenience tweaks

_(The Kobold)_ very good, i can see that would be hard to summarize

_(The Villain)_ things that sort your inventory better. things that automatically open the skills menu when you hit a target skill level you were aiming for

_(The Kobold)_ i might try it out

_(The Kobold)_ oh, that's nice

_(The Villain)_ 2 levels of "autofight protection" - a warning at 60% hp, and a forced stop at 40% hp. all values can be tweaked of course

_(The Villain)_ etc etc

**Comment by G0rg0** [comment]
I'm using this and it's quite good. Thanks for the work on it!
